Contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Auth is a simple kotlin project with one caveat: We're using a live Cogntio Userpool for integration tests and
+the values are provided using code generation at compile time.
+
+The build needs both `region` and `clientid` configured, either using our hashicorp vault cluster (obviously not accessible from the outside),
```

Comment:
`clientid` or `clientId` if the latter please compare and adjust the following text passages